includes/reports/overview/services/class-sensei-reports-overview-service-students.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * File containing the Sensei_Reports_Overview_Service_Students class.
+ *
+ * @package sensei
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Courses overview service class.
+ *
+ * @since 4.5.0
+ */
+class Sensei_Reports_Overview_Service_Students {
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {     }
+
+	/**
+	 * Get average grade of all lessons graded in all the courses filtered by students.
+	 *
+	 * @since 4.5.0
+	 * @access public
+	 *
+	 * @param array $user_ids user ids.
+	 * @return double $graded_lesson_average_grade Average value of all the graded lessons in all the courses.
+	 */
+	public function get_graded_lessons_average_grade_filter_users( $user_ids ) {
```

Comment:
Do you think it would be better to remove the `_filter_users` suffix from the method name?

Reply:
https://github.com/Automattic/sensei/pull/5091/commits/3113a7c7d1f550488bd444425249acc632da2c86